app/controllers/company/tickets_controller.rb

```diff
@@ -34,5 +44,13 @@ def read_user_units
     def ticket_params
       params.require(:ticket).permit(:name, :unit_id, :description)
     end
+
+    def ticket_resolution_params
+      params.require(:ticket).permit(:resolution)
+    end
+
+    def check_resolution_presence
+      params[:ticket][:resolution] != ''
```

Comment:
```suggestion
      params[:ticket][:resolution].present?
```

А взагалі це в моделі треба робити.